default.py

```diff
@@ -90,35 +89,35 @@ def play(self, item):
                     stream['url'] += '|%s=%s' % (header, stream['headers'][header])
             print 'Sending %s to player' % stream['url']
             li = xbmcgui.ListItem(path=stream['url'], iconImage='DefaulVideo.png')
-            
+
             sub = False
             if xbmcaddon.Addon('xbmc.addon').getAddonInfo('version') > "16":
                 sub = True
-                if stream['subs'] != None and stream['subs'] != '':
+                if stream['subs'] is None and stream['subs'] != '':
```

Comment:
asi spis: if stream['subs'] and stream['subs'] != '':
pripadne is not None

Reply:
sorry, to je samozrejme kravina. vim ma nejaky ten plugin na zvyraznovanie, editoval som to rucne.